README.md

```diff
@@ -1,7 +1,28 @@
-# FortiGate Autoscale Core module
-This is the Autoscale Core module written in TypeScript.
+# Autoscale Core
+The Core Library for the Fortinet Autoscale project.
 
-Note: This is a pre-release for internal development purposes.
+The Autoscale project provides a cloud-based multi-group auto scaling functionality for virtual machines of Fortinet products which have native HA features to form a cluster with failover protection.
+
+This project provides the fundamental, generic logics of a Hybrid Licensing architecture. It also includes implementations for certain popular cloud platforms.
+
+This project has the following features:
+ * Multi-group Hybrid Licensing models. (Hybrid, BYOL-Only, PAYG-Only)
+ * AWS Transit Gateway Integration (see: https://github.com/fortinet/fortigate-autoscale-aws)
+
+
+## Supported platforms
+This project supports autoscaling for the cloud platforms listed below:
+* Amazon AWS
+
+## NodeJS dependency
+
+To use this project as a dependency:
```

Comment:
Can it be used in any other way? If not, then specify that it must be used as a dependency.

Reply:
> Can it be used in any other way? If not, then specify that it must be used as a dependency.

No, it can't. Must be used as a NodeJS dependency. What change do you suggest here?

Reply:
From this comment I am understaning that this project must be used as a NodeJS dependency.
Is that correct?

Reply:
> From this comment I am understaning that this project must be used as a NodeJS dependency.
> Is that correct?

correct. it answers why the heading is NodeJS dependency.